traffic_ops/traffic_ops_golang/deliveryservicesregexes/deliveryservicesregexes.go

```diff
@@ -112,15 +112,14 @@ JOIN type as rt ON r.type = rt.id
 	if err != nil {
 		api.HandleErr(w, r, inf.Tx.Tx, http.StatusInternalServerError, nil, fmt.Errorf("getting accessible tenants for user - %v", err))
 	}
-	if len(where) > 0 {
+	if len(where) == 0 {
```

Comment:
This definitely shouldn't be reversed

Reply:
See [comment](https://github.com/apache/trafficcontrol/pull/4633#issuecomment-617996513), fixed in 003af41431.